### DISCUSSION TITLE
Photosync webdav upload returns Connection Reset

### DISCUSSION THREAD

**Discussion by pmatos** [photoprism/photoprism#1917] [Q&A] [answered]
Hi,

Since I recorded a large video that photosync on android doesn't work. It keeps saying "Connection reset". I wonder if there's some sort of parameter I need to tune on the photoprism end to keep the connection alive for these files to be uploaded.

Any suggestions?

Paulo Matos

**Comment by pmatos** [comment, marked as answer]
Wondering, since webdav connects via ssl, if this could be related to ssh keep-alive...

**Reply by graciousgrey** [reply]
As mentioned in the chat, increasing the upload size limit from nginx should solve this :)

**Reply by lastzero** [reply]
NGINX users are invited to contribute to our docs so other users don't run into the same issues: 
- https://github.com/photoprism/photoprism-docs/blob/master/docs/getting-started/proxies/nginx.md

**Comment by egodigitus** [comment]
Had the same error. Issue was a large video file on my client device with photosync. 
Solution for me was to edit the nginx.conf
```  
server {  
client_max_body_size 500M;  
}  
```  
@lastzero Added this change to the nginx example conf in the docs

**Reply by lastzero** [reply]
Should be merged and published. Thanks a lot!
